src/cargo/core/compiler/fingerprint/mod.rs

```diff
@@ -2124,6 +2131,9 @@ enum DepInfoPathType {
 ///
 /// The serialized Cargo format will contain a list of files, all of which are
 /// relative if they're under `root`. or absolute if they're elsewhere.
+///
+/// The `config_envs` argument is a set of environment variables that are
```

Comment:
I found it hard to text search this variable because the naming is not consistent. Do you have any specific reason calling it `config_envs` not `env_config`?

Reply:
I like `env_config` also.